### ISSUE TITLE
KChildArea within other container rendering causes misplaced nodes

### ISSUE THREAD

**Issue by NiklasRentzCAU** [kieler/KLighD#25]
In the current master, the following .kgt file causes a weird Layout:
```
kgraph hierarchicEdge
properties:
de.cau.cs.kieler.kgraphsynthesis.defaults = true
org.eclipse.elk.hierarchyHandling = INCLUDE_CHILDREN
org.eclipse.elk.direction = RIGHT
knode {
	krectangle {
		grid: columns = 1
		ktext("foo")
		krectangle {
			kchildArea
		}
	}
	knode src {
		kedge (-> dest:p_dest)
	}
	kport
}
knode dest {
	kport p_dest
}
```

 KLighD will show the graph like this in a diagram:
[childInContainer.png](https://rtsys.informatik.uni-kiel.de/jira/secure/attachment/15000/15000_childInContainer.png)

The edge should be a straight line leading into the destination port, not somewhere way above that. Also, the additional single port added to the left of the hierarchic node should be centered and not somewhere above the center.
Both these effects are affected by the offset of the child rendering within its node. Adding some linebreaks "\n" to the text above the renderings makes both these issues even worse.

**Comment by NiklasRentzCAU** [comment]
This issue is migrated from the old KIELER Pragmatics JIRA issue tracker. See the old discussion on this ticket in [KIPRA-1980](https://rtsys.informatik.uni-kiel.de/jira/projects/KIPRA/issues/KIPRA-1980).

**Comment by NiklasRentzCAU** [comment]
Fixed with #154
